polling_stations/apps/data_collection/management/commands/import_west_devon.py

```diff
@@ -17,4 +17,15 @@ def address_record_to_dict(self, record):
         ]:
             rec["accept_suggestion"] = False
 
-        return rec
+        if uprn == "10013755070":
+            rec["postcode"] = "EX201FQ"
+
+    def station_record_to_dict(self, record):
+
+        # Charter Hall (Polling Station No.3)
+        if record.polling_place_id == "6524":
+            record = record._replace(polling_place_uprn="10013752301")
+
+        # Milton Abbot Village Hall
+        if record.polling_place_id == "6502":
+            record = record._replace(polling_place_uprn="10001329795")
```

Comment:
As well as attaching the correct UPRN, can we also replace or remove the invalid/incorrect postcode here.
If we've got a sensible correction,apply it. If its still unclear, bin it.